package.json

```diff
@@ -4,7 +4,8 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "nodemon --exec babel-node home-task1.1.js"
```

Comment:
Add scripts also to run task1 and task2 as npm run task1 and npm run task2 respectively

Reply:
Added as suggested